NEWS.md

```diff
@@ -45,6 +45,7 @@ plumber 1.0.0.9999 Development version
 
 * Changed `future::plan()` from `multiprocess` to `multisession` in example API `14-future` as "Strategy 'multiprocess' is deprecated in future (>= 1.20.0)". (#747)
 
+* Fix Content-Length header on HTTP responses which forbid it (e.g. 1xx, 204, 304)
```

Comment:
Should probably be something more like

> Remove response bodies (and therefore the `Content-Length` header) for status codes which forbid it under the HTTP specification (e.g. 1xx, 204, 304). (#758, #760)

Reply:
Maybe you did, maybe you didn't. I wanted to point out that you could suggest code change directly using a suggestion block in your comment (```suggestion ... ```)
https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/commenting-on-a-pull-request

Then lazy me could click on a button and isn't that one the greatest achievement in human history? Click a button, works get done.